src/main/java/org/semver4j/internal/range/processor/TildeProcessor.java

```diff
@@ -26,10 +28,12 @@
  * </ul>
  */
 public class TildeProcessor implements Processor {
+    @NotNull
```

Comment:
That's not necessary in favor of field object assignment.

Reply:
Reasons 1 & 6